Add `--globby-options` flag (#6437)
stylelint/stylelint: docs/user-guide/usage/cli.md

Specify the formatter to format your results. [More info](options.md#formatter).

## `--globbyOptions, --go`

> CLI flags: `--globby-options, --go`
> 

Reviewer: [suggest] Can you please remove the redundant lines?

```suggestion
```

> CLI option to traverse dotfiles i.e Hidden folders/files will not be ignored. [More info](options.md#globbyOptions).

Reviewer: [suggest] Let's keep the URL hash lower-case.

```suggestion
CLI option to traverse dotfiles i.e Hidden folders/files will not be ignored. [More info](options.md#globbyoptions).
```

> CLI option to traverse dotfiles i.e Hidden folders/files will not be ignored. [More info](options.md#globbyoptions).

Reviewer: [suggest] How about making the description more generic?

```suggestion
Options in JSON format passed to [globby](https://github.com/sindresorhus/globby). [More info](options.md#globbyoptions).
```
